openedx/course-discovery
# feat: add a signal on update program

[PROD-2902](https://2u-internal.atlassian.net/browse/PROD-2902)
Adds a signal to be used in [taxonomy-connector](https://github.com/openedx/taxonomy-connector/pull/94) whenever the degree is updated based on overview.

Steps to verify:
1. set `FIRE_UPDATE_DEGREE_SKILLS_SIGNAL` to True
2. Publish a program through django admin
3. Notice a logged message similar to this: `Signal fired to update program skills. Program: [uuid]'`
